Menu/Menu.js

```diff
@@ -1,10 +1,35 @@
+let articlesAni = document.querySelector('.articles');
 
 const toggleMenu = () => {
-  // Toggle the "menu--open" class on your menu refence. 
+  // Toggle the "menu--open" class on your menu refence.
+  let menuValues = menu.classList.toString();
+  if(menuValues.search('menu--open') == -1){
+    menu.classList.add('menu--open');
+    articlesAni.classList.add('articlesOpacityIn');
+    articlesAni.addEventListener('click', () => {
+      menu.classList.add('menu--close');
+      articlesAni.classList.add('articlesOpacityOut');
+    });
+    menu.classList.remove('menu--close');
+    articlesAni.classList.remove('articlesOpacityOut');
+  } else {
+    menu.classList.add('menu--close');
+    articlesAni.classList.add('articlesOpacityOut');
+    articlesAni.classList.remove('articlesOpacityIn');
+    menu.classList.remove('menu--open');
+  }
 }
```

Comment:
You could use `element.classList.toggle("class-name")` here to simplify your code and not have to use an if-else statement.